.github/ISSUE_TEMPLATE/ask_question.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Ask a question
+about: Use this template to ask general questions about our SDK.
+title: ''
+labels: 'question'
+assignees: ''
+
+---
+
+## Description
+Please clearly state your question. Be as specific as possible to help us understand what you need assistance with. Provide some background on the situation or problem you're facing, explain what you are trying to achieve and where you encountered difficulties.
+
+## Code Snippets (if applicable)
```

Comment:
Let's use the same title and content here and in the bug report one. I personally prefer `Code Snippets` more than `SDK-Related Code Segments` but up to you